lib/components/first-time-point/style.less

```diff
@@ -90,4 +90,15 @@
       }
     }
   }
+
+  &@{prefix}no-arrow {
+
+    @{prefix}arrow {
```

Comment:
it seems very weird that you'd have a `dqpl-arrow` within a `dqpl-no-arrow` element...whats up with that?